python/lsst/ts/tunablelaser/ascii.py

```diff
@@ -167,7 +44,7 @@ class AsciiRegister:
 
     Parameters
     ----------
-    commander : `TCPIPClient`
+    component : `Laser`
```

Comment:
Please update the description of this parameter to reflect what it actually is. It is not a TCP/IP stream.